pkg/cli/tabular.go

```diff
@@ -1,20 +1,23 @@
 package cli
 
 import (
+	"encoding/json"
 	"fmt"
+	"sort"
 	"strings"
 
 	"github.com/charmbracelet/lipgloss"
 	"github.com/evertras/bubble-table/table"
+	"github.com/jeremywohl/flatten"
 	"github.com/spf13/cobra"
 )
 
 func NewTabular(rows ...[]string) table.Model {
 	columnKeyProperty := "Property"
 	columnKeyValue := "Value"
 	t := NewTable(
-		table.NewColumn(columnKeyProperty, columnKeyProperty, 37),
-		table.NewColumn(columnKeyValue, columnKeyValue, 37),
+		table.NewFlexColumn(columnKeyProperty, columnKeyProperty, 1),
```

Comment:
Does this still preserve our uuid minimum column size need to avoid `id` field truncation? If I understand from the docs correctly, this would give our property column to value column a 1:2 width ratio?

Reply:
Yikes. Let me look into that.